### PR TITLE
Get Pokemon Snap playable

### PR DESCRIPTION
**Pull request by purplemarshmallow**

### CODE REVIEW

**Comment by fzurita** [comment]
Can this work with frameBufferEmulation\copyToRDRAM=2?

**Comment by purplemarshmallow** [comment]
No, it need to be in sync mode here

**Comment by AmbientMalice** [comment]
Would it be possible to get a Zilmar test build of this? I imagine Mupen64 (not 64plus) will probably be able to emulate the red dot+everything else via partial copies.

**Comment by purplemarshmallow** [comment]
>I imagine Mupen64 (not 64plus) will probably be able to emulate the red dot+everything else via partial copies.

I only tested mupen64plus

File: src/FrameBuffer.cpp [hunk 1]
```diff
@@ -571,7 +571,7 @@ FrameBuffer * FrameBufferList::findTmpBuffer(u32 _address)
 
 void FrameBufferList::saveBuffer(u32 _address, u16 _format, u16 _size, u16 _width, u16 _height, bool _cfb)
 {
-	if (m_pCurrent != nullptr && config.frameBufferEmulation.copyAuxToRDRAM != 0) {
```
Comment (gonetz) [inline src/FrameBuffer.cpp]: Ok, leave it as is, if it helps with other games.

**Comment by gonetz** [comment]
Merged: 36fb0d180

Red dot works perfectly with Mupen64, but game hangs when course ended and you returned to lab.

**Comment by weinerschnitzel** [comment]
What about 1964?

**Comment by tony971** [comment]
Using Glupen64 with default settings, the red dot is now gone. However, the constant flickering and freeze at Oak's evaluation are also gone.

no pokemon

**Pull request closed without merging**